### DISCUSSION TITLE
Error when building my next.js app using vercel

### DISCUSSION THREAD

**Discussion by Yassin-Samir** [vercel/turborepo#8544] [Help]
### Summary

when i deploy my project to vercel and it start building. I get this error 
`Error: turbo_json_parse_error
  x Found an unknown key `tasks`.
   ,-[3:1]
`
although when i run it locally i get no errors. This is because the turborepo version is updated to latest in the vercel cli for no reasin and i don't know how to update it

**Comment by skycloud112** [comment]
I encountered the same error. After running codemon to upgrade to the new turbo version, I noticed that the turbo version in the root packages.json was still "^1.13.3". I resolved this by manually updating the turbo version to "2.0.5" in the root package.json.
